packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -213,7 +213,7 @@ export async function copyProjectTemplateAndReplace(
     },
     {
       id: options.useWinUI3 ? 'Microsoft.WinUI' : 'Microsoft.UI.Xaml',
-      version: options.useWinUI3 ? winui3Version : '2.3.191129002',
+      version: options.useWinUI3 ? winui3Version : '2.5.0-prerelease.201027002',
```

Comment:
could you please read this from the WinUI.props file instead of hardcoding it so that we have one less place to update next time? :)